event-sourcing/event-source/src/main/java/uk/gov/justice/services/eventsourcing/source/core/EventStreamManager.java

```diff
@@ -179,33 +171,41 @@ public UUID cloneAsAncestor(final UUID id) throws EventStreamException {
      * Clears the stream, deleting all associated events from the event_log, it does not update the
      * event_stream.
      */
-    public void clear(final UUID id) {
-        eventRepository.clear(id);
+    public void clearEventsForStream(final UUID id) {
+        eventRepository.clearEventsForStream(id);
     }
 
     /**
-     * Get the current (current maximum) sequence id (version number) for a stream
+     * Get the latest position number for a stream
      *
      * @param id the id of the stream
-     * @return the latest sequence id for the provided steam. 0 when stream is empty.
+     * @return the latest position number for the provided steam. 0 when stream is empty.
      */
-    public long getCurrentVersion(final UUID id) {
-        return eventRepository.getCurrentSequenceIdForStream(id);
+    public long getCurrentPosition(final UUID id) {
```

Comment:
This is confusingly named; it is actually getting the size of the stream. Maybe rename "getSize"?

Reply:
changed to getSize()